src/Microsoft.TemplateSearch.TemplateDiscovery/Filters/SkipTemplatePacksFilter.cs

```diff
@@ -21,7 +21,8 @@ internal sealed class SkipTemplatePacksFilter
             "microsoft.dotnet.wpf.projecttemplates",
             //NUnit package is included to SDK, however not managed by Microsoft - keep it in to check for updates
             //"nunit3.dotnetnew.template",
-            "microsoft.aspnetcore.components.webassembly.template"
+            "microsoft.aspnetcore.components.webassembly.template",
+            "microsoft.maui.templates"
```

Comment:
I think @baronfel has some more info on what we should do here. Should this be an exhaustive list of all templates in all optional workloads? (like we have Microsoft.Android.Templates, Microsoft.iOS.Templates, etc.) Or should we do this at all?

Reply:
@jonathanpeppers 
discussed with @baronfel - we still want to do it as a quick fix, and also include other packs coming from optional workloads, at least 3 mentioned, please let me know if you know more of them

In long term, we will automate it based on optional workloads manifest.

Reply:
I think the full list is:
```
microsoft.android.templates
microsoft.ios.templates
microsoft.maccatalyst.templates
microsoft.macos.templates
microsoft.maui.templates
microsoft.tvos.templates
```
And this should cover any template pack under the `maui` workload set. There might also be a `wasm` template in an optional workload? Maybe @lewing knows?